extensions/Persona/Config.pm

```diff
@@ -19,11 +19,6 @@ use constant REQUIRED_MODULES => [
         module  => 'JSON',
         version => 0,
     },
-    {
-        package => 'libwww-perl',
-        module  => 'LWP::UserAgent',
-        version => 0,
-    },
```

Comment:
So you're not going to like the response to this but BMO's CI breaks if I remove that code at the moment... even though it isn't used.